### PR TITLE
Feature/sergeyro/item list with avatar

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/3868111907

**TODO `AvatarListItem` props are not visible for some reason - don't know why**
image

### CODE REVIEW

File: src/components/ListItemAvatar/ListItemAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  avatarClassName?: string;
+}
+
+const ListItemAvatar: VibeComponent<ListItemAvatarProps> = forwardRef(
+  ({ className, id, src, avatarClassName }, ref) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+
+    return (
+      <div ref={mergedRef} className={cx(styles.listItemAvatar, className)} id={id} aria-hidden="true">
```
Comment (hadasfa) [inline src/components/ListItemAvatar/ListItemAvatar.tsx:19]: the aria hidden is cool!

File: src/components/ListItemIcon/ListItemIcon.tsx [hunk 2]
```diff
@@ -14,23 +14,23 @@ export interface ListItemIconProps extends VibeComponentProps {
   margin?: ListItemIconMargin;
 }
 
-const ListItemIcon: React.FC<ListItemIconProps> = forwardRef(
-  ({ className, id, icon, margin = ListItemIconMargin.START }, ref) => {
-    const componentRef = useRef(null);
-    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+const ListItemIcon: VibeComponent<ListItemIconProps> & {
```
Comment (hadasfa) [inline src/components/ListItemIcon/ListItemIcon.tsx:17]: It does contain all the required attributes for vibe component? I think not (no default test id).
I think we do not need to use this type here.

File: src/components/ListItemAvatar/ListItemAvatar.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { VibeComponent, VibeComponentProps } from "../../types";
+import Avatar from "../Avatar/Avatar";
+import styles from "./ListItemAvatar.module.scss";
+
+export interface ListItemAvatarProps extends VibeComponentProps {
+  src?: string;
+  avatarClassName?: string;
+}
+
+const ListItemAvatar: VibeComponent<ListItemAvatarProps> = forwardRef(
```
Comment (hadasfa) [inline src/components/ListItemAvatar/ListItemAvatar.tsx:13]: same comment about vibe component

**Review by hadasfa — APPROVED** [review]
Approved but let's talk about using the vibe component type for sub components

File: src/components/ListItemIcon/ListItemIcon.tsx (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/ListItemIcon/ListItemIcon.tsx:17]: It might contain defaultDataTestId, when we'll to test it, no?

**Pull request merged**